File: linearSearch.cpp
//g++  5.4.0

#include <iostream>
using namespace std;

int linearSearch (int array[], int length, int num){
    
    for (int i = 0; i < length; i++){
        if (num == array[i]){
           return i;
        }
    }
    
    return -1;
    
}
    
    
int main()
{
    int numbers[] = { 10, 20, 50, 60, 70, 80, 99};
    int length = sizeof(numbers)/ sizeof (numbers[0]);
    cout << linearSearch(numbers,length, 99);
}

File: linearSearch.php
<?php //php 7.0.8

function linearSearch($array, $num){
    
    $length = count($array);
    for ($i = 0; $i < count($array); $i++){
        if ($num == $array[$i]){
           return $i; 
        }
        
    }
    return -1;
}

 $numbers = array(10, 50, 60, 70, 80, 500, 1000, 55, 99, 90);
 echo (linearSearch($numbers, 99));
    
?>
